src/Application/Profiles/AppUserProfile.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Application.DTOs.AppUserDTO;
+using AutoMapper;
+using Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.Profiles
+{
+    public class AppUserProfile : Profile
+    {
+        public AppUserProfile()
+        {
+            CreateMap<AppUser, GetAppUserDTO>()
+                .ForMember(dest => dest.Id, src => src.MapFrom(x => x.Id))
```

Comment:
If I'm not mistaken, automapper automatically maps entity's properties to dto properties if they have the same type and name

Reply:
I requested to map explicit mapping